tidb: config tidb server labels (#10397)
pingcap/docs: follower-read.md

> **Note:**
>
> - When the value of `tidb_replica_read` is set to `closest-replicas` or `closest-adaptive`, each replica of the cluster should be distributed in each Region according to the specified configuration. To configure `location-labels` for PD and set the correct `labels` for TiDB and TiKV, refer to [Schedule replicas by topology labels](/schedule-replicas-by-topology-labels.md). TiDB depends on the `zone` label to match TiKV nodes in the same Region, so you need to make sure that the `zone` label is included in the `location-labels` of PD and `zone` is included in the configuration of each TiDB and TiKV node. If your cluster is deployed using TiDB Operator, refer to [Configure replicas across Regions](https://docs.pingcap.com/tidb-in-kubernetes/stable/configure-a-tidb-cluster#high-availability-of-data).

> > - When the value of `tidb_replica_read` is set to `closest-adaptive`, to prevent the unbalanced distribution of the read traffic in various Regions, TiDB dynamically detects whether the Region distribution of all online TiDB and TiKV nodes is balanced. If a Region contains only TiDB or TiKV nodes, TiDB forces to select the leader replica to perform read operations forcefully. For example, if all TiDB nodes in a Region hang, then other online TiDB nodes are downgraded to read using leader replicas. When there is at least one TiDB node in this Region is back online, TiDB continues to prefer to select the replica in the same Region to perform read operations.

Reviewer: ```suggestion
> - When the value of `tidb_replica_read` is set to `closest-adaptive`, to prevent the unbalanced distribution of the read traffic in various Regions, TiDB dynamically detects whether the Region distribution of all online TiDB and TiKV nodes is balanced. If a Region contains only TiDB or TiKV nodes, TiDB forces to select the leader replica to perform read operations forcefully. For example, if all TiDB nodes in a Region are down, then other online TiDB nodes are downgraded to read using leader replicas. After at least one TiDB node in this Region is back online, all TiDB nodes switch back to prefer to select the replica in the same Region to perform read operations.
```
